src/test/uint256_tests.cpp

```diff
@@ -73,54 +73,54 @@ inline uint160 uint160S(const std::string& str)
 
 BOOST_AUTO_TEST_CASE( basics ) // constructors, equality, inequality
 {
-    BOOST_CHECK(1 == 0+1);
+    BOOST_CHECK_EQUAL(1, 0+1);
```

Comment:
I wonder what this does exactly :)

Reply:
Indeed.. saw it too. Seems like temporary code that somehow made it into the test. Guess I could remove if I retouch.